.github/pr_management/common.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * Collection of common functions to use in this folder
+ */
+import * as core from '@actions/core'
+import * as github from '@actions/github';
+const token = core.getInput("repo-token");
+const octokit = github.getOctokit(token);
+
+const owner = github.context.repo.owner;
```

Comment:
I think its better if we abstract `github` into separate file e.g GithubManager or GithubHelper, so it will be easier to extend later